### ISSUE TITLE
Warnings with clang 6

### ISSUE THREAD

**Issue by mgerhardy** [karimnaaji/voxelizer#7]
```
voxelizer.h:793:49: warning: suggest braces around initialization of subobject [-Wmissing-braces]
                            vx_triangle_t t1 = {v1, v2, boxcenter, {0.0f, 0.0f, 0.0f}};
                                                ^~~~~~~~~~~~~~~~~
                                                {                }
voxelizer.h:794:49: warning: suggest braces around initialization of subobject [-Wmissing-braces]
                            vx_triangle_t t2 = {v2, v3, boxcenter, {0.0f, 0.0f, 0.0f}};
                                                ^~~~~~~~~~~~~~~~~
                                                {                }
voxelizer.h:795:69: warning: suggest braces around initialization of subobject [-Wmissing-braces]
                            vx_triangle_t t3 = {v3, v1, boxcenter, {0.0f, 0.0f, 0.0f}};
                                                                    ^~~~~~~~~~~~~~~~
                                                                    {               }
voxelizer.h:842:23: warning: suggest braces around initialization of subobject [-Wmissing-braces]
    vx_vertex_t vs = {voxelsizex, voxelsizey, voxelsizez};
                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                      {                                 }
voxelizer.h:902:21: warning: suggest braces around initialization of subobject [-Wmissing-braces]
    vx_vec3_t vs = {voxelsizex, voxelsizey, voxelsizez};
                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                    {                                 }
```

**Comment by karimnaaji** [comment]
@mgerhardy ah thanks for the heads up, i'll fix that

**Comment by mgerhardy** [comment]
fixed with #8
